sktime/performance_metrics/tests/test_performance_metrics_forecasting.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+
+__author__ = ["Tomasz Chodakowski"]
+
+import pytest
+
+from sktime.performance_metrics.forecasting import mape_loss
+from sktime.performance_metrics.forecasting import smape_loss
+
+from sktime.performance_metrics.tests._config import TEST_YS, TEST_YS_ZERO
+
+
+@pytest.mark.parametrize("test_y", TEST_YS_ZERO)
+def test_all_losses_near_zero(test_y):
```

Comment:
I'd either rename the function to `test_mape_near_zero` or add the other losses to the function.

Reply:
Thanks Markus, I have changed the name to test_mape_loss_near_zero but will revisit it to introduce near zero checks for all losses as this would be helpful. However, there is small discrepancy between the way sMAPE/MASE and MAPE handle such situation. The former return nan when encountering 0 denominator while MAPE returns large value (that comes from division by a small value - machine epsilon). The second approach seem pragmatic and taken by scikit learn as well but happy to change that behaviour, either way, to be consistent. What the view here?

Reply:
I haven't thought much about this, but following scikit-learn seems sensible.

Reply:
OK. Lets get this PR in and I will do next one with proposed changes to sMAPE (handle special case where y_test and y_pred are both 0 -> return 0 instead of nan) and MASE (special case where reference, in-sample mae is 0 -> return relatively large error instead of nan) this should make it consistent and follow the approach in scikit learn